oap-server/server-bootstrap/src/main/resources/otel-oc-rules/istio-controlplane.yaml

```diff
@@ -28,7 +28,7 @@
 #    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
 #    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
 # </pre>
-expSuffix: tag({tags -> tags.cluster = 'istio-ctrl::' + tags.cluster}).service(['cluster', 'app'])
```

Comment:
Does this make sense @hanahmily ? Seems both control plane and data plane all share the same "serviceGroup" in "oap-server/server-bootstrap/src/main/resources/ui-initialized-templates/istio.yml"

Reply:
No, we should let the OT collector handle it by "relabel" as the OAP cluster is a center cluster that can't identify the metrics from which cluster.

Reply:
This rule file intends to ingest metrics from different Kubernetes clusters, not identify control plane and data plane. There might be more than one cluster when two different OT collectors transfer metrics to the OAP cluster.